GhostNetwork.Content/Comments/ICommentsService.cs

```diff
@@ -13,6 +13,8 @@ public interface ICommentsService
 
         Task<(DomainResult, string)> CreateAsync(string key, string text, string replyCommentId, UserInfo author);
 
+        Task<(DomainResult, (Comment, int))> UpdateAsync(string commentId, string content);
```

Comment:
Too complicated return type. It will be better to return `Task<(DomainResult, Comment)>` or even `Task<(DomainResult)>` if you don't need to return updated model in http response

Reply:
Fix it, you're right, i don't need to return updated comment